egs/csj/s5/local/chain/tuning/run_tdnn_1b.sh

```diff
@@ -0,0 +1,235 @@
+#!/usr/bin/env bash
+
+# This is a spec augmentation experiment.
+
+# steps/info/chain_dir_info.pl exp/chain/tdnn1b
+# exp/chain/tdnn1b: num-iters=2610 nj=2..2 num-params=13.6M dim=40+100->3920 combine=-0.075->-0.075 (over 23) xent:train/valid[1737,2609,final]=(-0.949,-0.924,-0.922/-0.950,-0.935,-0.932) logprob:train/valid[1737,2609,final]=(-0.066,-0.063,-0.063/-0.072,-0.071,-0.071)
+
+# local/chain/compare_wer.sh --online exp/chain/tdnn1b
+# System                        tdnn1a    tdnn1b
+# WER eval1                      10.30     10.05
+#          [online:]             10.30     10.05
+# WER eval2                       8.59      8.09
+#          [online:]              8.56      8.07
+# WER eval3                       9.90      9.33
+#          [online:]              9.90      9.31
+# Final train prob             -0.0532   -0.0628
+# Final valid prob             -0.0776   -0.0708
+# Final train prob (xent)      -0.8289   -0.9225
+# Final valid prob (xent)      -0.9539   -0.9319
```

Comment:
I tried some neural network structures referring to other recipes, but WER wasn't improved.
Only the spec augmentation slightly improved WER.

Reply:
A difference between run_tdnn_1a.sh and run_tdnn_1b.sh is shown below.
```
# diff run_tdnn_1a.sh run_tdnn_1b.sh
< affix=1a  # affix for the TDNN directory name
---
> affix=1b  # affix for the TDNN directory name
47c44
< num_epochs=4
---
> num_epochs=10
53,54c50,51
< num_jobs_initial=3
< num_jobs_final=10
---
> num_jobs_initial=2
> num_jobs_final=2
145a143,145
>   idct-layer name=idct input=input dim=40 cepstral-lifter=22 affine-transform-file=$dir/configs/idct.mat include-in-init=true
>   batchnorm-component name=batchnorm0 input=idct include-in-init=true
>   spec-augment-layer name=spec-augment freq-max-proportion=0.5 time-zeroed-proportion=0.2 time-mask-max-frames=20 include-in-init=true
```